tests/ui/symbol-names/const-generics-structural-demangling.rs

```diff
@@ -43,25 +45,31 @@ pub struct TupleByteBool<const TBB: (u8, bool)>;
 //~| ERROR demangling-alt(<c::TupleByteBool<{(1, false)}>>)
 impl TupleByteBool<{(1, false)}> {}
 
-pub struct OptionUsize<const OU: Option<usize>>;
+#[derive(PartialEq, Eq, ConstParamTy)]
```

Comment:
Likewise, I could add `ConstParamTy` derive to `Option` too... I just made it a local type since it's not really the point of the test (afaict?) and I didn't bother.

Reply:
Seems fine to me, its probably for the best to avoid putting `ConstParamTy` on various std types atm